cluster/cluster_config.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
```

Comment:
Having a script or template for the headers could be convenient. It seems it's manually managed.